package-overrides/github/Eonasdan/bootstrap-datetimepicker@4.7.14.json

```diff
@@ -0,0 +1,27 @@
+{
+  "format": "global",
+  "main": "build/js/bootstrap-datetimepicker.min.js",
```

Comment:
Can we use the unminified sources? It is good to do this for the best development experience.